server/routes/oauth.js

```diff
@@ -1,18 +1,29 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const passport = require('passport');
-const { loginUser } = require('../controllers/oauth');
-const isLoggedIn = require('../middleware/isLoggedIn');
+const passport = require("passport");
+const {
+  loadUser,
+  logOut,
+  loginUser,
+  doesUserExist,
+} = require("../controllers/oauth");
+const isLoggedIn = require("../middleware/isLoggedIn");
 
-router.get('/auth/google',
-    passport.authenticate('google', { scope : ['email', 'profile'] }));
+router.post("/user", doesUserExist);
 
-router.get('/google/callback',
-    passport.authenticate('google', {
-        successRedirect: '/oauth/login',
-}));
+router.get(
+  "/google",
+  passport.authenticate("google", { scope: ["email", "profile"] }, loginUser)
+);
 
-router.route('/login').get(isLoggedIn, loginUser);
+router.get(
+  "/google/callback",
+  passport.authenticate("google", {
+    successRedirect: `${process.env.CALEND_APP_DEV_URL}/dashboard`,
```

Comment:
_failureRedirect_, please handle this too

Reply:
May I know if there is a reason why conflicts are still unresolved?

Reply:
Hi, I wasn't aware there were merge conflicts but I just resolved them!